lib/asm-parser.js

```diff
@@ -27,9 +27,13 @@ const _ = require('underscore'),
     AsmRegex = require('./asmregex').AsmRegex;
 
 class AsmParser extends AsmRegex {
-    constructor(compilerProps) {
+    constructor(compilerProps, labelDef = undefined, commentOnly = /^\s*(((#|@|;|\/\/).*)|(\/\*.*\*\/))$/, removeBlankLines = false) {
```

Comment:
would prefer it if this could be part of an override, but I know this class is awkward and it might not be possible...